x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/condition_entry_input/index.tsx

```diff
@@ -62,14 +69,9 @@ export const ConditionEntry = memo<ConditionEntryProps>(
     onVisited,
     'data-test-subj': dataTestSubj,
   }) => {
-    const getTestId = useCallback(
-      (suffix: string): string | undefined => {
-        if (dataTestSubj) {
-          return `${dataTestSubj}-${suffix}`;
-        }
-      },
-      [dataTestSubj]
-    );
+    const getTestId = useCallback((suffix: string) => dataTestSubj && `${dataTestSubj}-${suffix}`, [
```

Comment:
hmm... is this doing what it was doing before?  will `undefined` be returned if `dataTestSubj` is not defined? I'm thinking it will now return `false` which is not valid value.

Reply:
<img width="1023" alt="test" src="https://user-images.githubusercontent.com/1155504/100764937-368bc380-33f7-11eb-8998-e3b851685675.png">

Reply:
Logical operators return one of the values used in operator, it just treated as truthy/falsy during evaluation.